src/shared/reducers/settings.js

```diff
@@ -25,7 +25,8 @@ export function settingsByArchiveId(state = {}, action) {
 
 const DEFAULT_SETTINGS = {
   columnSizes: { tree: 230, entries: 230 },
-  condencedSidebar: true
+  condencedSidebar: true,
+  menubarVisibility: true
```

Comment:
Yeah.. I think `menubarIsVisible` or `menubarVisible` might make more sense as it's boolean. What do you feel?

Reply:
@perry-mitchell I named it after the method that sets it, but yeah we can change it. `menubarVisible` sounds good